config/content/Hero.js

```diff
@@ -14,13 +14,41 @@ export default {
     alt: 'Howdy buddy! We are back with another awesome virtual hackathon, HackNITR',
   },
   ticker: [
-    { id: 1, src: '/hero-icons/amazon.svg', alt: 'amazon logo' },
-    { id: 2, src: '/hero-icons/apple.svg', alt: 'apple logo' },
-    { id: 3, src: '/hero-icons/dell.svg', alt: 'dell logo' },
-    { id: 4, src: '/hero-icons/google.svg', alt: 'google logo' },
-    { id: 5, src: '/hero-icons/nasa.svg', alt: 'nasa logo' },
-    { id: 6, src: '/hero-icons/paypal.svg', alt: 'paypal logo' },
-    { id: 7, src: '/hero-icons/tata.svg', alt: 'tata logo' },
-    { id: 8, src: '/hero-icons/tesla.svg', alt: 'tesla logo' },
+    {
+      id: 1,
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/alpha/vlnvmirmr4r7tolmdrxt.png',
+      alt: 'Devfolio',
+      size: '150px',
+    },
+    {
+      id: 2,
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/alpha/oigy7pgs2haeoh8dtqon.png',
+      alt: 'Github',
+      size: '120px',
+    },
+    {
+      id: 3,
+      src: 'https://res.cloudinary.com/dalqfvowk/image/upload/project-oregano/sponsors/alpha/ogaeyl6batos4n5cumtz.png',
```

Comment:
Very large size image. Use squoosh.app

Reply:
> Very large size image. Use squoosh.app

Yeah, I'll squoosh the large sized logos and update it in cloudinary🚀
Thanks for reviewing it @DesignrKnight <3